go/systests/ephemeral_test.go

```diff
@@ -357,11 +357,12 @@ func readdToTeamWithEKs(t *testing.T, leave bool) {
 	teamID, teamName := user1.createTeam2()
 	user1.addTeamMember(teamName.String(), user2.username, keybase1.TeamRole_WRITER)
 
-	ephemeral.ServiceInit(user1.tc.G)
-	ekLib := user1.tc.G.GetEKLib()
-
-	_, err := ekLib.GetOrCreateLatestTeamEK(context.Background(), teamID)
-	require.NoError(t, err)
+	for _, u := range tt.users {
```

Comment:
since user1 is the only one doing operations on the team only they need the `ServiceInit` call